source/sync/data-model/update-data-model/update-schema-production-apps.txt

```diff
@@ -1,20 +1,430 @@
+
+- C++ SDK - unsupported
+- Flutter SDK v2.1.0
+- Java SDK v10.18.0
+- Kotlin SDK -
+- .NET SDK v11.7.0
+- Node.js SDK v12.6.0
+- React Native SDK v12.6.0
+- Swift SDK v10.46.0
+
+.. _update-data-model-production-app-procedure:
+
+Make Changes with Schema Versioning
+-----------------------------------
+
+The following describes how to make changes to your server-side schema through
+the App Services UI when schema versioning is enabled. For information on the
+specific changes you can make to your schema, refer to
+:ref:`sync-schema-overview`.
+
+.. procedure::
+
+   .. step:: Make Schema Changes in App Services UI
+
+      In the App Services UI, navigate to the :guilabel:`Schema` page.
+
+      1. From the :guilabel:`Collections` tab, select the schema that you want
+         to modify.
+
+      .. note:: Define Default Values When Possible
+
+         If your changes impact the default values of a property (for example,
+         when adding a new required property), we recommend that you define a
+         default value. To learn how, refer to
+         :ref:`schema-update-add-a-default-value`.
+
+      2. Do one of the following, depending on whether you are using deployment
+         drafts in your app:
+
+         - If drafts are disabled, click :guilabel:`Save`.
+         - If drafts are enabled, click :guilabel:`Save Draft`. From the
+           banner, click :guilabel:`Review Draft & Deploy`.
+
+      4. If you *only* made non-breaking changes, your changes are automatically
+         applied to the current schema version without any additional handling.
+         A new schema version is *not* created.
+
+         .. note:: Added Elements are Not Available to Previous Versions
+
+            Because additive changes are applied to the current schema, clients
+            must use a schema version equal to or greater than the
+            current version to reference any elements added as part of
+            your changeset.
+
+      5. If you made at least one schema versioning or breaking change, a
+         confirmation window displays with a summary of your changes.
+
+   .. step:: Confirm and Deploy Changes
+
+      From the confirmation window, review the :guilabel:`Schema Version` and
+      :guilabel:`Change Summary` information:
+
+      - The current schema version, and the incremented number of the
+        new schema created upon deployment (if applicable).
+      - The type of change, labeled as ADDITIVE, SUPPORTED, or UNSUPPORTED.
+      - A description of the changes to the noted object type or field.
+      - Any notes or suggested actions to take for a given change. For example,
+        if App Services is expecting a default value based on the change you
+        made, the note suggests that you define a default value.
+
+      How you deploy your changes depends on the changes in your changeset:
+
+      .. tabs::
+
+         .. tab:: Schema Versioning
```

Comment:
If you adopt the suggestion I made above, these tabs would become "Partially-Compatible" and "Incompatible".